index.bs

```diff
@@ -16,69 +16,73 @@ Indent: 2
 Markup Shorthands: css no, markdown yes
 Version History: https://github.com/w3ctag/security-questionnaire/commits/master/index.bs
 !Bug Reports: <a href="https://github.com/w3ctag/security-questionnaire/issues/new">via the w3ctag/security-questionnaire repository on GitHub</a>
-Abstract:
-  This document provides help in considering the privacy impact of
-  a new feature or specification as well as common mitigation strategies for
-  common privacy impacts. The questions are meant to be useful when considering
-  the security and privacy aspects of a new feature or specification and the
-  mitigation strategies are meant to assist in the design of the feature or
-  specification. Authors of a new proposal or feature should implement the
-  mitigations as appropriate; doing so will assist in addressing the respective
-  points in the questionnaire. Given the variety and nature of specifications,
-  it is likely that the listed questions will not be comprehensive in a way
-  enabling to reason about the full privacy impact, and some mitigations may
-  not be appropriate or other mitigations may be necessary. It is nonetheless
-  the aim to present the questions and mitigations as a starting point, helping
-  to consider security and privacy at the start of work on a new feature, and
-  throughout the lifecycle of a feature.
-
-  It is not meant as a "security checklist", nor does an editor or group’s use
-  of this questionnaire obviate the editor or group’s responsibility to obtain
-  "wide review" of a specification security and privacy properties before
-  publication. Furthermore, the filled questionnaire should not be understood
-  as security and privacy considerations, although part of the answers may be
-  relevant in drafting the considerations.
+Abstract: This document contains a set of questions to be used when evaluating the security and privacy characteristics of web platform technologies.
 </pre>
 
 <!-- Big Text: Intro -->
 <section>
   <h2 id="intro">Introduction</h2>
 
-  New features make the web a stronger and livelier platform. Throughout the feature
-  development process there are both foreseeable and unexpected security and privacy
-  risks. These risks may arise from the nature of the feature, some of its part(s),
-  or unforeseen interactions with other features.  Such risks and may be mitigated
-  through careful design and application of security and privacy design patterns.
-
-  Standardizing web features presents unique challenges. Descriptions, protocols
-  and algorithms need to be considered strictly before they are broadly adopted by
-  vendors with large user bases.  If features are found to have undesirable privacy
-  properties after they are standardized, then, browser vendors may break compatibility
-  in their implementations to protect users' privacy as the user agent is the user’s agent.
-
-  *This is why each Working Group needs to consider security and privacy by design and by default*.
```

Comment:
And why do we remove the clear way of saying that security and privacy should be considered by default? :)